coalib/results/TextPosition.py

```diff
@@ -32,6 +32,54 @@ def line(self):
     def column(self):
         return self._column
 
+    def __lt__(self, other):
+        """
+        Test whether ``self`` is behind the other
+        ``TextPosition``.
+
+        If the column in a ``TextPosition`` is ``None``, consider
+        whole line. If the line in a ``TextPosition`` is ``None``,
+        consider whole file.
+
+        :param other: ``TextPosition`` to compare with.
+        :return:      Whether this ``TextPosition`` is behind the other
+                      one.
+        """
+        if self.line is None or other.line is None:
+            return False
+        if self.line == other.line:
+            return (False
```

Comment:
Sorry for the late reply.

How about:

```python
if a:
    return True

if next_condition:
    return something_else

# and as many conditions as you want - the first one that's true will return control to the calling function

return final
```

But if this is the only thing holding up this PR, please go ahead and merge it.